dist_tables/dml.rst

```diff
@@ -38,6 +38,70 @@ For example:
 
     If COPY fails to open a connection for a shard placement then it behaves in the same way as INSERT, namely to mark the placement(s) as inactive unless there are no more active placements. If any other failure occurs after connecting, the transaction is rolled back and thus no metadata changes are made.
 
+INSERT INTO ... SELECT
```

Comment:
True, the "Real-time event aggregation at scale" blog post also distributes on tenant_id. I think I was just trying to simplify the example by omitting the tenant stuff and going with an entity id approach.

I can sketch an idea for a diagram in our docs sync-up and see what we think.